src/Components/HomeCards/HomeCards.tsx

```diff
@@ -32,7 +35,7 @@ function HomeCards() {
           <path d="M256 112c-35.3 0-64 28.7-64 64s28.7 64 64 64 64-28.7 64-64-28.7-64-64-64zm0 110.9c-25.9 0-46.9-21-46.9-46.9s21-46.9 46.9-46.9 46.9 21 46.9 46.9-21 46.9-46.9 46.9z" />
         </svg>
       </div>
-      <div className={styles.singlecard}>
+      <div className={styles.singlecard} onClick={() => navigate("/online")}>
```

Comment:
You could also use a Link here instead of navigate... 
Navigate is normally used when you cannot pass the navigation prop into the component directly like Login